src/main.css

```diff
@@ -8,9 +8,8 @@
 }
 
 html {
-    background: #00162d;
-    box-sizing: border-box;
-    color: #f6f6f6;
+    background: #282c37;  /* fallback for old browsers */
```

Comment:
Fallback for old browsers?

Reply:
I was testing gradients and that was a fallback if it wasn't supported. Didn't remove the comment :P